### PR TITLE
Ws 019 admin endpoint

### PR DESCRIPTION
**Pull request by mtharkins**
Add boolean to Sighting model & endpoint to Admin controller for unapproved sightings

### CODE REVIEW

**Comment by mtharkins** [comment]
Sorry I had missed the merge conflict previously, it's been resolved now and is ready for review.

File: WhaleSpotting/Controllers/AdminController.cs [hunk 1]
```diff
@@ -1,15 +1,29 @@
 using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
+using SQLitePCL;
```
Comment (AndrewYHuang) [inline WhaleSpotting/Controllers/AdminController.cs:3]: Probably want to remove this

**Review by AndrewYHuang — CHANGES_REQUESTED** [review]
Just one small change, then it should be ready to merge in!

File: WhaleSpotting/Controllers/AdminController.cs (hunk 1 above)
Comment (AndrewYHuang) [reply WhaleSpotting/Controllers/AdminController.cs:3]:
```suggestion
```

**Review by AndrewYHuang — APPROVED** [review]

**Pull request merged**